tests/pyspark_delta/test_pyspark_delta.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+import chispa
+import pyspark
+from delta import *
+
+builder = (
+    pyspark.sql.SparkSession.builder.appName("MyApp")
+    .config("spark.sql.extensions", "io.delta.sql.DeltaSparkSessionExtension")
+    .config(
+        "spark.sql.catalog.spark_catalog",
+        "org.apache.spark.sql.delta.catalog.DeltaCatalog",
+    )
+)
+
+spark = configure_spark_with_delta_pip(builder).getOrCreate()
```

Comment:
Expensive, reusable pieces of infrastructure should leverage pytest.fixtures, so that the context is created once and injected in all tests that need it.  and it's already there, by the way, here

https://github.com/delta-incubator/dat/blob/master/tests/conftest.py#L7

Reply:
Yea, what you have looks better, but can we take out the `spark.stop` so the tests run faster?

Reply:
the finalizer is another fixture that is executed by pytest when all tests are run, see https://docs.pytest.org/en/6.2.x/reference.html#std-fixture-request